dropshot/src/server.rs

```diff
@@ -914,17 +932,11 @@ async fn http_request_handle<C: ServerContext>(
             let request_log = rqctx.log.clone();
             let worker = server.handler_waitgroup_worker.clone();
             let handler_task = tokio::spawn(async move {
-                let request_log = rqctx.log.clone();
                 let result = handler.handle_request(rqctx, request).await;
 
                 // If this send fails, our spawning task has been cancelled in
-                // the `rx.await` below; log such a result.
-                if tx.send(result).is_err() {
```

Comment:
I'd like to keep a log message here to help debug cross-system issues.  (It can be useful to know when the server finished doing something, even if the client long ago gave up, so you can put together a timeline of events.)  How about something like `"request completed after handler was already cancelled"`?  I wonder if we can include the result here, too -- status code and/or error message?

Reply:
Sounds good, I've kept the log and added the status code and error messages to it.